### ISSUE TITLE
Binance BLVT max position size

### ISSUE THREAD

**Issue by xmatthias** [ccxt/ccxt#9996]
- OS: linux
- Programming Language version: python
- CCXT version:
- 
Based on the [binance anouncement](https://www.binance.com/en/support/announcement/6a1d2aedfd9c453591916c533fd26aeb), Binance now enfoces an upper position limit on leveraged coins (~5000$).

```python
import ccxt
exchange = ccxt.binance()
lm = load_markets()
print(lm['LTCUP/USDT'])
```

The data for max-position (in the below example for LTCUP) can be found in MAX_POSITION - and is 1145 LTCUP.

![2021-09-17-125130_361x294_scrot](https://user-images.githubusercontent.com/5024695/133771025-9e406e3d-739a-4f92-96e7-22ebb69934c1.png)

The Binance UI also confirms this part.

ccxt's uniform structure (non-'info') does not contain this limit whatsoever.
While #9987 suggests a new error if the limit is exceeded (eventually by multiple buys) - there's little help provided in preventing this in the first place.
In my opinion, the entry for maximum lot-size is wrong (it's showing up as 920_000 - while the maximum lot-size one can buy in one order is 1145) - so i think `limits.amount.max` should be using the minimum between `max_position_size` and `LOT_SIZE.maxqty`, otherwise `limits.amount.max` is missleading me that i can buy 920_000 LTCUP - while that'd exceed my maximum position size by a large factor.

On top of that, an additional "max_position_size" might (!) be added to the "limits" section - but it's not absolutely necessary (at least not for my current usage).

```
{'percentage': True,
 'feeSide': 'get',
 'tierBased': False,
 'taker': 0.001,
 'maker': 0.001,
 'precision': {'base': 8, 'quote': 8, 'amount': 2, 'price': 3},
 'limits': {'amount': {'min': 0.01, 'max': 920000.0},      <-- This seems pretty high, the max position size is 1145
  'price': {'min': 0.437, 'max': 8.284},
  'cost': {'min': 10.0, 'max': None},
  'market': {'min': 0.0, 'max': 47446.73765277}},
 'id': 'LTCUPUSDT',
 'lowercaseId': 'ltcupusdt',
 'symbol': 'LTCUP/USDT',
 'base': 'LTCUP',
 'quote': 'USDT',
 'baseId': 'LTCUP',
 'quoteId': 'USDT',
 'info': {'symbol': 'LTCUPUSDT',
  'status': 'TRADING',
  'baseAsset': 'LTCUP',
  'baseAssetPrecision': '8',
  'quoteAsset': 'USDT',
  'quotePrecision': '8',
  'quoteAssetPrecision': '8',
  'baseCommissionPrecision': '8',
  'quoteCommissionPrecision': '8',
  'orderTypes': ['LIMIT',
   'LIMIT_MAKER',
   'MARKET',
   'STOP_LOSS_LIMIT',
   'TAKE_PROFIT_LIMIT'],
  'icebergAllowed': True,
  'ocoAllowed': True,
  'quoteOrderQtyMarketAllowed': False,
  'isSpotTradingAllowed': False,
  'isMarginTradingAllowed': False,
  'filters': [{'filterType': 'PRICE_FILTER',
    'minPrice': '0.43700000',
    'maxPrice': '8.28400000',
    'tickSize': '0.00100000'},
   {'filterType': 'PERCENT_PRICE',
    'multiplierUp': '5',
    'multiplierDown': '0.2',
    'avgPriceMins': '5'},
   {'filterType': 'LOT_SIZE',
    'minQty': '0.01000000',
    'maxQty': '920000.00000000',
    'stepSize': '0.01000000'},
   {'filterType': 'MIN_NOTIONAL',
    'minNotional': '10.00000000',
    'applyToMarket': True,
    'avgPriceMins': '5'},
   {'filterType': 'ICEBERG_PARTS', 'limit': '10'},
   {'filterType': 'MARKET_LOT_SIZE',
    'minQty': '0.00000000',
    'maxQty': '47446.73765277',
    'stepSize': '0.00000000'},
   {'filterType': 'MAX_NUM_ORDERS', 'maxNumOrders': '200'},
   {'filterType': 'MAX_POSITION', 'maxPosition': '1145.00000000'},
   {'filterType': 'MAX_NUM_ALGO_ORDERS', 'maxNumAlgoOrders': '5'}],
  'permissions': ['LEVERAGED']},
 'spot': True,
 'type': 'spot',
 'margin': False,
 'future': False,
 'delivery': False,
 'linear': False,
 'inverse': False,
 'expiry': None,
 'expiryDatetime': None,
 'active': True,
 'contractSize': None}
```

**Comment by frosty00** [comment]
`ccxt.binance` contains only information about spot markets, you will need to use `ccxt.binanceusdm` to loadMarkets of futures symbols

**Comment by xmatthias** [comment]
'LTCUP/USDT' is a spot-market (it's a leveraged pair - but it's trade on spot-markets, not futures/margin markets).

**Comment by ndubel** [comment]
> In my opinion, the entry for maximum lot-size is wrong (it's showing up as 920_000 - while the maximum lot-size one can buy in one order is 1145) - so i think `limits.amount.max` should be using the minimum between `max_position_size` and `LOT_SIZE.maxqty`, otherwise `limits.amount.max` is missleading me that i can buy 920_000 LTCUP - while that'd exceed my maximum position size by a large factor.

it's not the best solution because if you already have 500 LTCUP you can buy maximum 645 LTCUP. Because 1145 it's something like "max hold".  So I think that this way is preffered:

> On top of that, an additional "max_position_size" might (!) be added to the "limits" section - but it's not absolutely necessary (at least not for my current usage).

**Comment by xmatthias** [comment]
yes i don't disagree - but i think both ways should be done (now i also think the reported "max-lot-size" from binance itself is wrong - as they declare a lot-size that can't be traded) - but that's beyond the scope of this issue i think.

**Comment by xmatthias** [comment]
Closing this as Binance stopped supporting BLVT in april 2024 . . . [anouncement](https://www.binance.com/en/support/announcement/detail/b982e8502e0d4a42aec8191a878f7895).
